src/tabs/__integ__/tabs.test.ts

```diff
@@ -302,3 +315,89 @@ test(
     await expect(page.isFocused(wrapper.findTabContent().toSelector())).resolves.toBe(true);
   })
 );
+
+test(
+  'verifies focus moves to dismissible via keyboard',
+  setupTest(async page => {
+    await page.focusTabHeader();
+    await page.click(wrapper.findTabLinkByIndex(3).toSelector());
+    await page.keys(['ArrowRight']);
+    // Since the focus doesn't actually update on the action/dismissible buttons,
```

Comment:
I was trying to use the `page.isDisplayed/isFocused` assertion to evaluate focus here, and `isFocused` evaluates to false in the case of action/dismissible since they don't invoke the `onChange` event to set focus to a tab.

The test here validates you can navigate to dismissible as part of the tab stop (unless we don't think it's needed)